managers/oclc_catalog.py

```diff
@@ -96,17 +96,20 @@ def _get_other_editions(self, oclc_number: int, offset: int=0):
             )
 
             if other_editions_response.status_code != 200:
-                logger.warning(f'OCLC other editions request failed with status {other_editions_response.status_code}')
+                logger.warning(
+                            f"Other editions query for no {oclc_number} failed "
+                            f"With status {other_editions_response.status_code} "
+                            f"and reason {other_editions_response.json()['type']}")
```

Comment:
I think we should be careful with getting the json body. If it's a 500 error, there may not be a json body and we may error out here.